Layers/LayerConfigurator.gd

```diff
@@ -1,21 +1,23 @@
 extends Configurator
 
+const SQLite = preload("res://addons/godot-sqlite/bin/gdsqlite.gdns")
 
 var geodataset
 var center
+var geopackage
 
 
 func _ready():
 	set_category("geodata")
-	add_test_data()
+	digest_geopackage()
 
 
-# Adds static test data; will be removed as soon as we have a valid GeoPackage.
-func add_test_data():
-	var geopackage_path = get_setting("gpkg-path")
+# Digests the information provided by the geopackage
+func digest_geopackage():
+	var geopackage_path = get_setting("gpkg-path") + ".db"
```

Comment:
Look above :) 
2shady4u/godot-sqlite#55

Reply:
Oh right, missed that! They actually just published a release of 3.1 at the GitHub repo a few days ago - I added that to the LandscapeLab and it seems to be working as intended. I'll push the upgrade so that we're rid of the `.db`

Reply:
perfect